lib/checker.rb

```diff
@@ -0,0 +1,25 @@
+module SlackApi
+  class Checker
```

Comment:
This class is all about wrapping incoming Slack request data into an object, so maybe a better name for this class could be simply `Request` rather than `Checker`?